### ISSUE TITLE
Can not cover code in process.on('exit')

### ISSUE THREAD

**Issue by popomore** [gotwarlost/istanbul#567]
File called `exit.js`

```js
'use strict';

process.on('exit', function() {
  console.log('can\'t cover');
});

function cover() {
  console.log('can cover');
}
cover();
```

Then run it

```
istanbul cover exit.js
```

So the result is no cover.

<img width="320" alt="screenshot 2016-03-10 1 03 11" src="https://cloud.githubusercontent.com/assets/360661/13643563/78f4ac52-e65c-11e5-9ab0-4ec3b42ad7f9.png">

**Comment by gotwarlost** [comment]
Yeah, that's because istanbul's process.exit handler runs before yours. Not easy to solve but shouldn't be impossible either. Need to look into it.
